openpype/hosts/fusion/plugins/publish/collect_render.py

```diff
@@ -4,7 +4,10 @@
 
 from openpype.pipeline import publish
 from openpype.pipeline.publish import RenderInstance
-from openpype.hosts.fusion.api.lib import get_frame_path
+from openpype.hosts.fusion.api.lib import (
+    get_frame_path,
+    get_current_comp,
+)
```

Comment:
Revert to old code, since we'll need to remove the use of `get_current_comp` here - see other comment.